manifests/init.pp

```diff
@@ -120,7 +120,7 @@
   simplib::assert_metadata($module_name)
 
   if $enable != 'ignore' {
-    if $use_firewalld {
+    if ( 'firewalld' in pick($facts['simplib__firewalls'], 'none') ) and $use_firewalld {
```

Comment:
shouldn't this be `[ 'none' ]`

Reply:
@lnemsick-simp It'll return either the Array from `$facts['simplib__firewalls']` or the String `none`.

Reply:
@lnemsick-simp Ah...well, I was wrong. It does indeed pick the first entry from the Array :-/.

This means that some other code needs to be fixed as well.

Reply:
@lnemsick-simp OK, I had a bad test. `pick()` does work as expected in this case.